app/controllers/hearings_controller.rb

```diff
@@ -16,7 +16,7 @@ class HearingsController < ApplicationController
   def show
     add_breadcrumb "#{t('generic.hearing_day')} #{hearing_day&.strftime('%d/%m/%Y')}", ''
 
-    return if hearing
+    return if @hearing
```

Comment:
Does this need to be an instance variable?

Reply:
I used the instance variable because there is also a hearing method, this hearing method is what's used for the v1 call, whereas the @hearing instance variable is the result of either the v1 or v2 call depending on the flag